core/web/jobs_controller.go

```diff
@@ -59,7 +60,7 @@ func (jc *JobsController) Show(c *gin.Context) {
 	jobSpec := job.Job{}
 	if externalJobID, pErr := uuid.FromString(c.Param("ID")); pErr == nil {
 		// Find a job by external job ID
-		jobSpec, err = jc.App.JobORM().FindJobByExternalJobID(c.Request.Context(), externalJobID)
+		jobSpec, err = jc.App.JobORM().FindJobByExternalJobID(externalJobID, pg.WithParentCtx(c.Request.Context()))
```

Comment:
I'm curious how come you don't want to use a request context across layers?
Let's say if http request timeout is 5 secs, but postgres query takes more than 5 sec, exceeded context will just cancel this query. It is a good practice to use a context there. Am I missing something?

Reply:
Using the request context across layers is precisely what we are doing here. If the request context is cancelled all queries will be cancelled. I assumed this would be the desired behaviour. Perhaps I misunderstood the question?

Reply:
Ah, sorry, my bad, missed some parts :) Thanks!